modules/groupmanager-ffg.js

```diff
@@ -251,15 +251,18 @@ export class GroupManager extends FormApplication {
 
   async _rollTable(table, type) {
     let r = new Roll("1d100").roll();
-    let rollOptions = game.settings.get("starwarsffg", "privateTriggers") ? {rollMode: "gmroll"} : {};
-    r.toMessage({
-      flavor: `${game.i18n.localize("SWFFG.Rolling")} ${type}...`,
-    }, rollOptions);
+    let rollOptions = game.settings.get("starwarsffg", "privateTriggers") ? { rollMode: "gmroll" } : {};
+    r.toMessage(
+      {
+        flavor: `${game.i18n.localize("SWFFG.Rolling")} ${type}...`,
+      },
+      rollOptions
+    );
     let filteredTable = table.filter((entry) => entry.rangeStart <= r.total && r.total <= entry.rangeEnd);
-    let tableResult = filteredTable?.length ? `${filteredTable[0].type} ${type} ${game.i18n.localize("SWFFG.Triggered")} ${game.i18n.localize("SWFFG.For")} ${filteredTable[0].name}` : `${game.i18n.localize("No")} ${type} ${game.i18n.localize("SWFFG.Triggered")}`;
+    let tableResult = filteredTable?.length ? `${filteredTable[0].type} ${type} ${game.i18n.localize("SWFFG.Triggered")} ${game.i18n.localize("SWFFG.For")} @Actor[${filteredTable[0].playerId}]{${filteredTable[0].name}}` : `${game.i18n.localize("No")} ${type} ${game.i18n.localize("SWFFG.Triggered")}`;
```

Comment:
Real change is just on this line. Realized I didn't have prettier set in VSCode when I made earlier changes so it's auto-formatting my commits now

Reply:
Just noticed that the `No` localization is incorrect and should be  `SWFFG.OptionValueNo`

Reply:
I was trying to use one of the core localizations not a system localization

Reply:
I'll go ahead and switch to a system localization next time I get to my computer

Reply:
Done